ontology/yaml/resources/TRANSPORT/entity_types/ABSTRACT.yaml

```diff
@@ -22,4 +22,12 @@ SS:
   opt_uses:
   - failed_alarm
   - lost_power_alarm
+  - fire_alarm
 
+
+EPM:
+  description: "Elevator position monitoring"
+  is_abstract: true
+  uses:
+  - floor_status
+  - elevator_status
```

Comment:
Do `elevator_status` and `floor_status` go hand in hand? Meaning would you expect that anytime we have a `floor_status` we would also have an `elevator_status`? And that both of these are critical to position monitoring? If not it may be better to make `elevator_status` optional.

Reply:
fixed thank you